comp/metadata/internal/util/inventory_payload.go

```diff
@@ -57,6 +57,7 @@ import (
 	"fmt"
 	"path/filepath"
 	"sync"
+	"sync/atomic"
```

Comment:
I think there is a linter preventing using `sync/atomic` in favor of an equivalent from `uber-go/atomic`, you might get a lint error due to that

Reply:
Indeed !